tests/testLinearizeLookupTable.py

```diff
@@ -203,7 +203,7 @@ def makeDetector(self, bbox=None, numAmps=None, rowInds=None, colIndOffsets=None
                 ampInfo.setName("amp %d_%d" % (i + 1, j + 1))
                 ampInfo.setBBox(boxArr[i, j])
                 ampInfo.setLinearityType(linearityType)
-                ampInfo.setLinearityCoeffs([rowInds[i, j], colIndOffsets[i, j], 0, 0])
+                ampInfo.setLinearityCoeffs(np.array([rowInds[i, j], colIndOffsets[i, j], 0, 0]))
```

Comment:
I don't think so. `rowInds[i, j]` is an int, `colIndOffsets[i, j]` is a float, the others are ints. My guess is that swig got more strict about type conversions.